packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@woocommerce/dependency-extraction-webpack-plugin",
-	"version": "1.5.0",
+	"version": "1.6.0",
```

Comment:
DEWP will also need to be bumped here
https://github.com/woocommerce/woocommerce-admin/blob/14e3ace58b19c71b55469fb1ee0c5e7e65a3e6f4/bin/starter-pack/_package.json#L25